pkg/controller/environment.go

```diff
@@ -265,20 +258,11 @@ func (env *K8sEnvironment) PrepareRun(stopCh <-chan struct{}) error {
 		func(obj interface{}) bool {
 			return cont.handleQosPolUpdate(obj)
 		}, stopCh)
```

Comment:
Shouldnt the Qos queue be processed after the qosInformer has run in qos.go?

Reply:
Yes, we need to move this

Reply:
I have moved the function.